packages/strapi-plugin-upload/config/schema.graphql

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  resolver: {
+    Query: {
+      file: false,
+      files: {
+        resolver: 'Upload.find'
+      }
+    }
+  }
+};
```

Comment:
File to delete ? Or let it for examples.

Reply:
No, we need it to make GraphQL working with the Upload plugin.